backend/vkt/src/main/java/fi/oph/vkt/view/ExamEventXlsxView.java

```diff
@@ -0,0 +1,183 @@
+
+    final List<ClerkEnrollmentDTO> enrollments = examEvent
+      .enrollments()
+      .stream()
+      .sorted(enrollmentComparator())
+      .toList();
+
+    for (int i = 0; i < enrollments.size(); i++) {
+      final Row row = sheet.createRow(i + 1);
+      final ClerkEnrollmentDTO enrollment = enrollments.get(i);
+      int ci = 0;
+      row.createCell(ci).setCellValue(DATE_FORMAT.format(examEvent.date()));
+      row.createCell(++ci).setCellValue(examEvent.language().name());
+      row.createCell(++ci).setCellValue(DATETIME_FORMAT.format(enrollment.enrollmentTime()));
+      row.createCell(++ci).setCellValue(enrollment.person().lastName());
+      row.createCell(++ci).setCellValue(enrollment.person().firstName());
+      row.createCell(++ci).setCellValue(enrollment.person().identityNumber());
+      formatNullableDate(row.createCell(++ci), enrollment.previousEnrollmentDate());
+
+      row.createCell(++ci).setCellValue(statusToText(enrollment.status()));
+
+      formatBoolean(row.createCell(++ci), enrollment.oralSkill());
+      formatBoolean(row.createCell(++ci), enrollment.textualSkill());
+      formatBoolean(row.createCell(++ci), enrollment.understandingSkill());
+
+      formatBoolean(row.createCell(++ci), enrollment.readingComprehensionPartialExam());
+      formatBoolean(row.createCell(++ci), enrollment.speechComprehensionPartialExam());
+      formatBoolean(row.createCell(++ci), enrollment.writingPartialExam());
+      formatBoolean(row.createCell(++ci), enrollment.speakingPartialExam());
+
+      row.createCell(++ci).setCellValue(enrollment.email());
+      row.createCell(++ci).setCellValue(enrollment.phoneNumber());
+
+      formatBoolean(row.createCell(++ci), enrollment.digitalCertificateConsent());
+      row.createCell(++ci).setCellValue(enrollment.street());
+      row.createCell(++ci).setCellValue(enrollment.postalCode());
+      row.createCell(++ci).setCellValue(enrollment.town());
+      row.createCell(++ci).setCellValue(enrollment.country());
+    }
+
+    autoresizeExcelColumns(sheet, headers);
+  }
+
+  private static Comparator<ClerkEnrollmentDTO> enrollmentComparator() {
+    final Comparator<ClerkEnrollmentDTO> byStatus = Comparator.comparing(ClerkEnrollmentDTO::status);
+    final Comparator<ClerkEnrollmentDTO> byEnrollmentTime = Comparator.comparing(ClerkEnrollmentDTO::enrollmentTime);
+    return byStatus.thenComparing(byEnrollmentTime);
+  }
+
+  private static void formatNullableDate(final Cell cell, final LocalDate localDate) {
+    if (localDate != null) {
+      cell.setCellValue(DATE_FORMAT.format(localDate));
+    } else {
+      cell.setBlank();
+    }
+  }
+
+  private static void formatBoolean(final Cell cell, final boolean b) {
+    if (b) {
+      cell.setCellValue(1);
+    } else {
+      cell.setBlank();
+    }
+  }
+
+  private static String statusToText(final EnrollmentStatus status) {
+    switch (status) {
+      case PAID -> {
+        return "Maksettu";
+      }
```

Comment:
```suggestion
      case PAID -> "Maksettu";
```

En tiiä onko prettierin sääntö tai joku muu best practice mutta tässä varmaan vois lyhentää näin. Sinänsä mitätön juttu mutta piti yrittää jotain kommenttia keksiä 😄

Reply:
Muutettu, ei siinä mitään ollut, tehdessä vaan vänkäsi vastaan niin en jäänyt taistelemaan sen kanssa ja sit jäi korjaamatta.